share/spice/just_delete_me/content.handlebars

```diff
@@ -3,13 +3,8 @@
     {{notes}}
   </p>
 {{/if}}
-{{#if delete_url}}
```

Comment:
Without the `Instructions: ` some of the text can be mi
![delete_account_plenty_of_fish_at_duckduckgo](https://cloud.githubusercontent.com/assets/873785/14222004/cb755cd0-f839-11e5-9124-d2b5685d34fd.png)
sleading: